package.json

```diff
@@ -14,7 +14,8 @@
     "devDependencies": {
         "@types/d3": "5.7.2",
         "d3": "5.12.0",
-        "powerbi-visuals-api": "^3.8.3",
+        "powerbi-visuals-api": "~5.1.0",
+        "powerbi-visuals-utils-formattingmodel": "~5.0.0",
         "powerbi-visuals-tools": "^3.3.2",
```

Comment:
Please, update dependencies.

Reply:
what do you mean? they are updated here to 5.1.0 and 5.0.0

Reply:
`powerbi-visuals-tools` has new version